spec/Section 1 -- Overview.md

```diff
@@ -27,13 +27,14 @@ The GraphQL Composite Schemas specification has a number of design principles:
   specification facilitates the coordinated effort of combining collaboratively
   designed source schemas into a single coherent composite schema.
 
-- **Evolvable**: A _composite schema_ offers an integrated, product-centric API
-  interface to clients. Each source _GraphQL service_ that underpins the
-  composite schema should be able to evolve without disrupting these clients.
-  Over time, the same functionality may be provided by a different combination
-  of services, while the composite schemas interface must continue to support
-  existing requests from all clients; source schema boundaries are therefore
-  considered an implementation detail and should never be exposed to clients.
+- **Evolvable**: A _composite schema_ enables offering an integrated,
+  product-centric API interface to clients. Each source _GraphQL service_ that
+  underpins the composite schema should be able to evolve without disrupting
+  these clients. Over time, the same functionality may be provided by a
+  different combination of services, while the composite schemas interface can
+  continue to support existing requests from all clients; source schema
+  boundaries are therefore considered an implementation detail and should never
```

Comment:
I think "should not" rather than "should never" is better here. I know this was in the original text, but while we're at it...

```suggestion
  boundaries are therefore considered an implementation detail and should not
```